README.md

```diff
@@ -33,7 +33,19 @@ described in detail below.
 
 Sign up for IBM's [Data Science Experience](http://datascience.ibm.com/). By signing up for the Data Science Experience, two services: ``DSX-Spark`` and ``DSX-ObjectStore`` will be created in your Bluemix account.
 
-## 4. Create the notebook
+## 2. Create the notebook
+
+Create the Project:
+* From the [IBM Data Science Experience page](https://apsportal.ibm.com/analytics) either click the "Projects" tab at the top or scroll down to "Recently updated projects".
```

Comment:
we need to figure out which url is better.  datascience.ibm.com is what I'd assumed, but this one works nicely